doc/specs/OOBE-Dialog.md

```diff
@@ -0,0 +1,169 @@
+| **Run on startup** | ![N](../images/OOBE/N.png "No") | By default, this is already on. For new users, this is probably not something we want to show as they might turn it off prematurely without understanding the value it provides. |
+| **Always Run as Admin** | ![N](../images/OOBE/N.png "No") | This is not a critical option for most use cases and more of a nuance that general users will not be concerned about. |
+
+#### 3.2.2. Color Picker
+
+| Option | Include? | Why |
+| --- | --- | --- |
+| **Activate Color Picker (Shortcut)** | ![Y](../images/OOBE/Y.png "Yes") | This is a crucial setting for utilizing Color Picker that users should have a say on. |
+| **Activation Behavior** | ![Y](../images/OOBE/Y.png "Yes") | This is critical for users to set up their experience the way they want. |
+| **Color Format for Clipboard** | ![Y](../images/OOBE/Y.png "Yes") | This is especially useful as it exposes the various copy options that may otherwise go overlooked. |
+| **Enable Color Picker** | ![N](../images/OOBE/N.png "No") | We want customers to use Color Picker first before they decide to disable it. Showing this option likely decreases the chance of that happening. |
+| **Editor Color Formats** | ![N](../images/OOBE/N.png "No") | This is typically an extraneous customization option for most users and would likely not be critical in a first-run experience. |
+
+#### 3.2.3. FancyZones
+
+| Option | Include? | Why |
+| --- | --- | --- |
+| **Hold Shift key to activate zones while dragging** | ![Y](../images/OOBE/Y.png "Yes") | This is a crucial operation that many first-time users might overlook if they do not read the documentation as noted in [#7421](https://github.com/microsoft/PowerToys/issues/7241#issuecomment-723561408). |
+| **Launch Zones Editor (Shortcut)** | ![Y](../images/OOBE/Y.png "Yes") | This is a crucial setting for utilizing FancyZones that users should have a say on. |
+| **Everything Else** | ![N](../images/OOBE/N.png "No") | These settings are typically extraneous customization options that most users would not find useful in a first-run experience. |
+
+#### 3.2.4. File Explorer
+
+| Option | Include? | Why |
+| --- | --- | --- |
+| **Everything** | ![N](../images/OOBE/N.png "No") | This gets tricky to incorporate as it requires admin privileges. In addition, many first-run users may not find value in the available options (or even understand what the options mean). A good description/visual should suffice for this page. |
+
+#### 3.2.5. Image Resizer
+
+| Option | Include? | Why |
+| --- | --- | --- |
+| **Everything** | ![N](../images/OOBE/N.png "No") | Many first-run users will not find value in these options (or even understand what the options mean).A good description/visual should suffice for this page. |
```

Comment:
That's not true. I know some people who are using PT especially to resize images. I would implement a picture that shows that you find image resizer in the cintext menu

Reply:
This is about showing any Image Resizer settings. I think what's meant here is that there are no specific Image Resizer settings that are super important to show in OOBE.

Reply:
Good point, I'll clarify.